express/public/index.html

```diff
@@ -0,0 +1,118 @@
+<!DOCTYPE html>
+<html lang="ko">
+  <head>
+    <meta charset="UTF-8" />
+    <title>coupang</title>
+    <link rel="stylesheet" href="./public/style/scss/main.css" />
+  </head>
+  <body>
+    <header class="header-box">
+      <!-- 최상단 메뉴 -->
+      <div class="header-top-bar">
+        <menu class="top-bar-favorite">
```

Comment:
낯설어서 검색했더니..
' The <menu> tag was deprecated in HTML 4.01 and again included in HTML 5.1 specification. But it will better to ignore it as it is experimental and not supported by many browsers.'

https://www.javatpoint.com/html-menu-tag#:~:text=HTML%20tag%20specifies%20a,elements%20within%20it.